AuxTel/Troubleshooting/General-Troubleshooting/AuxTel-Lost-Pointing-Accuracy.rst

```diff
@@ -0,0 +1,129 @@
+.. Review the README in this procedure's directory on instructions to contribute.
+.. Static objects, such as figures, should be stored in the _static directory. Review the _static/README in this procedure's directory on instructions to contribute.
+.. Do not remove the comments that describe each section. They are included to provide guidance to contributors.
+.. Do not remove other content provided in the templates, such as a section. Instead, comment out the content and include comments to explain the situation. For example:
+	- If a section within the template is not needed, comment out the section title and label reference. Include a comment explaining why this is not required.
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *I. Sotuela*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *Carlos Morales, Erik Dennihy*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _AuxTel-AuxTel-Troubleshooting-General-Troubleshooting-AuxTel-Lost-Pointing-Accuracy-Procedure:
+.. Each section should include a label for cross-referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with a reST object such as a title or figure, you must include the link with an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+#############################
+AuxTel Lost Pointing Accuracy
+#############################
+
+.. _AuxTel-Lost-Pointing-Accuracy-Procedure-Overview:
+
+Overview
+========
+
+This procedure should be used when the AuxTel loses pointing accuracy, causing targets to not appear centered in the detector during the first acquisition.
+
+.. _AuxTel-Lost-Pointing-Accuracy-Procedure-Error-Diagnosis:
+
+Error diagnosis
```

Comment:
Question: 2 weeks ago we used this procedure when we encounter the error: (RuntimeError: Telescope timed out getting in position; RuntimeError: Centroid finding algorithm was unsuccessful) Should it be mentioned in the document?, maybe it’s obvious or there are more error that this procedure applies.

Reply:
The "RuntimeError: Centroid finding algorithm was unsuccessful" is generally caused by something in the image, as the algorithm is not able to find a centroid, due to a double bright sources or missing totally a bright source in the image. In the case you mentioned, it seems like the telescope timed-out and the image didn´t contain a bright source for pointing, or WEP. This procedure worked because it recovered the pointing, but won´t always be the case with that mentioned error.

Reply:
Thanks Iona